WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -23,41 +21,34 @@
 
 import java.util.List;
 
-public class PeopleManagementActivity extends AppCompatActivity {
-
-    private PeopleAdapter mPeopleAdapter;
+public class PeopleManagementActivity extends AppCompatActivity implements PeopleListFragment.OnPersonSelectedListener {
+    private static final String KEY_PEOPLE_LIST_FRAGMENT = "people-list-fragment";
+    private static final String KEY_PERSON_DETAIL_FRAGMENT = "person-detail-fragment";
 
     @Override
-    public void onCreate(Bundle savedInstanceState) {
+    protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
-        int localBlogId = BlogUtils.getBlogLocalId(WordPress.getCurrentBlog());
-        Blog blog = WordPress.getBlog(localBlogId);
-
         ActionBar actionBar = getSupportActionBar();
         if (actionBar != null) {
             actionBar.setHomeButtonEnabled(true);
             actionBar.setDisplayHomeAsUpEnabled(true);
         }
         setContentView(R.layout.people_management_activity);
-
         setTitle(R.string.people);
```

Comment:
Again not part of this changeset but, since the Activity title is not dynamic, may I suggest setting it via the manifest, like we do elsewhere in the app? Cheers!